mod/gcforums/views/default/admin/administer_utilities/debugging_forums.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+
+$title = elgg_echo('Debugging the GCconnex Forums');
+$dbprefix = elgg_get_config('dbprefix');
+
+
+
+$query = "	SELECT * FROM elggentity_relationships r 
```

Comment:
These will need to use the dbprefix var above

Reply:
removed the administrative utilities option